scripts/alltools.tcl

```diff
@@ -437,3 +445,17 @@ proc ordnumber {string} {
   }
   return $string
 }
+
+proc ucwords {string} {
+	list asent
+	foreach word [split [join $string]] {
+		lappend asent [string totitle $word]
+	}
+	set string [join $asent]
+	return $string
+}
+
+proc uclwords {string} {
+	set string [ucwords [string tolower [join $string]]]
```

Comment:
This `join` isn't right, `[string tolower $string]` is correct on its own. Join requires a list as argument